Makefile

```
@@ -23,6 +23,43 @@ build-server:
 		-X ${REPO}/pkg/version.BuildDate=${BUILDTIME}" \
 		-o ./bin/server ./cmd/server;
 
+build-mobile-server-for-debugging:
+	go build -ldflags "-X ${REPO}/pkg/version.Version=${VERSION} \
+		-X ${REPO}/pkg/version.Revision=${REVISION} \
+		-X ${REPO}/pkg/version.Branch=${BRANCH} \
+		-X ${REPO}/pkg/version.BuildUser=${BUILDUSER} \
+		-X ${REPO}/pkg/version.BuildDate=${BUILDTIME}" \
+		-o ./bin/debug-mobile ./cmd/debug-mobile
+
+build-and-start-mobile-server-for-debugging: build-mobile-server-for-debugging
+	./bin/debug-mobile
+
+prepare-build-ios-for-debugging:
+	ionic build
+	make bindings-ios
+	npx cap sync
+
+build-ios-for-debugging: prepare-build-ios-for-debugging
+	npx cap open ios
+	$(info ************ Build app now in IDE and launch simulator ************)
+
+build-ios-for-live-debugging: prepare-build-ios-for-debugging
+	$(info ************ Build app now in IDE and launch simulator ************)
+	ionic capacitor run ios -l --address=0.0.0.0
+
+prepare-build-android-for-debugging:
+	ionic build
+	make bindings-android
+	npx cap sync
+
+build-android-for-debugging: prepare-build-android-for-debugging
+	npx cap open android
+	$(info ************ Build app now in IDE and launch simulator ************)
+
+build-android-for-live-debugging: prepare-build-android-for-debugging
+	$(info ************ Build app now in IDE and launch simulator ************)
+	ionic capacitor run android -l --address=0.0.0.0
```

I'm not a big fan of having the execution commands in the Makefile, maybe we can just add sth. like

```suggestion
build-ionic:
	ionic build
	npx cap sync

build-android: bindings-android build-ionic

build-ios: bindings-ios build-ionic

build-mobile: bindings-android bindings-ios build-ionic
```

---

Suggested change implemented.